x-pack/plugins/ml/public/embeddables/anomaly_swimlane/anomaly_swimlane_embeddable_factory.tsx

```diff
@@ -235,6 +237,19 @@ export const getAnomalySwimLaneEmbeddableFactory = (
         anomalySwimLaneServices
       );
 
+      const fetchSubscription = fetch$(api)
```

Comment:
You don't need a dedicated subscription. Try
```suggestion
      subscriptions.add(fetch$(api)
```